sql/2020/22_HTTP_2/percent_of_sites_affected_by_cdn_prioritization_issues.sql

```diff
@@ -0,0 +1,30 @@
+# standardSQL
+# % of sites affected by CDN prioritization issues (H2 and served by CDN)
+SELECT 
+  client,
+  IF(pages.cdn = "", "Not using CDN", pages.cdn) AS CDN,
+  IF(prioritization_status IS NOT null, prioritization_status, "Unknown") prioritizes_correctly,
+  COUNT(0) AS num_pages,
+  ROUND(COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client), 4) AS pct
+FROM (
+    SELECT
+      date,
+      client,
+      url,
+      JSON_EXTRACT_SCALAR(payload, "$._cdn_provider") as cdn
```

Comment:
cdn_provider is available as a column so don't need to extract this manually yourself.

Reply:
Makes sense! Will make the change

Reply:
Change made and committed